spec/routing/bulletins_routing_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+describe 'routes for bulletins', type: :routing do
+  describe 'GET /api/v1/bulletins/:id' do
+    let(:id) { 1 }
+    let(:show_bulletin) { get "/api/v1/bulletins/#{id}" }
+
+    it 'routes to bulletins#show(id)' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.